app/models/customization.rb

```diff
@@ -0,0 +1,3 @@
+# Convenience class alias to prevent carpal tunnel syndrome
+# see: app/models/jurisdiction_template_version_customization.rb
+Customization = JurisdictionTemplateVersionCustomization
```

Comment:
lol we should maybe think about name-spacing models.. I might try [this](https://blog.bullettrain.co/rails-model-namespacing/) with step code and see how it works

Reply:
OK keep me posted